Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -390,7 +390,7 @@ const ScrollView = React.createClass({
 
   _handleScroll: function(e: Object) {
     if (__DEV__) {
-      if (this.props.onScroll && !this.props.scrollEventThrottle && Platform.OS === 'ios') {
+      if (this.props.onScroll && this.props.scrollEventThrottle == null && Platform.OS === 'ios') {
```

Comment:
`== null` returns true for both null and undefined so this is fine.

According to the doc:
```
The default value is zero, which results in the scroll event being sent only once each time the view is scrolled
```
I agree the default should probably be 16 but I'd rather not make a breaking change for this.